contrib/guix/README.md

```diff
@@ -437,9 +437,8 @@ In the extraordinarily rare case where you messed up your Guix installation in
 an irreversible way, you may want to completely purge Guix from your system and
 start over.
 
-1. Uninstall Guix itself according to the way you installed it. (e.g. `sudo apt
-   purge guix` for Ubuntu packaging, `sudo make uninstall` for
-   built-from-source).
+1. Uninstall Guix itself according to the way you installed it (e.g. `sudo apt
```

Comment:
should it be partial due to missing changes in `doc/build-openbsd.md` (depends on bitcoin/bitcoin#22335_